recipes/configs/mistral/7B_lora.yaml

```diff
@@ -0,0 +1,69 @@
+# This config is currently a WIP. Use it with caution
```

Comment:
Are these configs based on the paper and/or other results showing good performance?

Reply:
Mostly digging on various forums like reddit